recipe/meta.yaml

```diff
@@ -55,11 +53,8 @@ test:
     - nbformat
     - numpy >=1.23
     - pandas
-    - pygments
-    - pytest-cov
     - trio
     {% endif %}
-
   commands:
```

Comment:
does `MIGRATING=false run_test.py` work (and removing the migrating stuff elsewhere)? Maybe with a note about what the migrating thing means and syncing with conda-forge?

Reply:
I found the whole migrating stuff overcomplicated

Reply:
If I recall it's for their python buildout. ipython is a somewhat central dependency which you need built before the test dependencies it uses.

Reply:
The way our buildout was done, this mechanism was not required.